lib/src/model.dart

```diff
@@ -632,66 +627,25 @@ class Class extends ModelElement
         .toList(growable: false);
   }
 
-  List<Method> get allInstanceMethods {
-    if (_allInstanceMethods != null) return _allInstanceMethods;
-    _allInstanceMethods = []
-      ..addAll([]
-        ..addAll(instanceMethods)
-        ..sort(byName))
-      ..addAll([]
-        ..addAll(inheritedMethods)
-        ..sort(byName));
-    return _allInstanceMethods;
-  }
+  Iterable<Method> get allInstanceMethods => quiverIterables.concat([instanceMethods, inheritedMethods]);
```

Comment:
You might/could use `instanceMethods.followedBy(inheritedMethods)` – I think

Reply:
@kevmoo actually replacing these with an expand call would work just as well I think.  There's nothing that special going on in quiver's concat.

Reply:
I'm not sure followedBy works very well for cases where there are more than two; a lot of FollowedByIterables start getting nested and that seems not as good in my reading of the implementation.